### ISSUE TITLE
[Not a bug] Proposal to set up GitHub Discussions

### ISSUE THREAD

**Issue by glmlm** [nvim-lua/kickstart.nvim#1285]
I have been looking at the Issue Tracker for this project over the past few months and there seems to be a certain segment of the community that seems to think that it is a free support forum for users.
To maintain the quality of the Issue Tracker, it would be preferable to direct these kinds of issues to the forum and allow users to resolve them among themselves.
I understand the sensitivity of this matter, but I would appreciate your input.

**Comment by iton0** [comment]
**TL;DR**: While I agree with the idea of adding Discussions, I think the core issue is that new users aren't reading documentation. Two areas to address: improving the issue template to prompt users to review docs, and increasing awareness of Kickstart documentation.

---

I agree with the sentiment, but I have hesitation and two concerns that might be contributing to the issue:

1. **Improve the issue template**: Add a checklist to confirm if users have read the docs. This would encourage research and help collaborators identify those who’ve put in the effort to read the issue template.
  
2. **Review Kickstart documentation**: The issue isn’t necessarily poor documentation but lack of exposure. We need to find ways to increase awareness of the Kickstart documentation.

By addressing these points, we can better evaluate whether adding Discussions is the right approach.

**Comment by feoh** [comment]
@glmlm I know where you're coming from here. I've often thought that having Discussions where I could ask questions about a project could be helpful to me as an end user.

But try on a different pair of shoes for a moment if you will. I am the only one with merge privs still paying attention with any regularity here, and we have a few regular contributors who help me understand proposed changes where I feel out of my depth.

First, I don't have the power to do what you're asking, you'd need to petition the repo owners, who don't have the time to participate here much..

Anyway, I'm not necessarily averse to this if others feel it's the right fit for this project as well, and if we can get the Powers that Be to enable them.

I just know I don't have any more time to contribute to this particular project than I spend already :)

**Comment by feoh** [comment]
Given what I just wrote, I'm going to close this for now. Conditions that would make re-opening make sense are:

- Broad interest in the community
- Finding a way to get folks with the power to enable this to do so

Until then, I'm closing this. Thanks for understanding.

**Comment by iton0** [comment]
@feoh I think it may be appropriate to take another look at this and get @tjdevries input about adding Discussions as @glmlm suggested. Particularly, I think doing an approach similar to [ghostty](https://github.com/ghostty-org/ghostty/issues) would be more effective compared to the checklist that I suggested.

**Comment by feoh** [comment]
@iton0 I'm not against any of it. He doesn't check in here very often and I am just a co-maintainer so I have exactly *zero* power in this situation and if I'm honest, I'm *still* not convinced that opening up a discussion thingie wouldn't just be another time sink for maintainers who don't have much as it is :)

He's on Twitter, I'm pretty sure his email is on Github. If you feel strongly, by all means contact him. I have no feelings on the matter and I don't feel like this is the best use of my time.

Not trying to be mean, and I REALLY appreciate all the effort you put into helping me review changes, I just don't have any real control at the level that would be required and don't feel nearly strongly enough to go above and beyond on this.

**Comment by iton0** [comment]
No worries, I completely get it. I’m not trying to put any pressure on you, just wanted to let you know where I stand. I’ll look into reaching out directly to him—thanks for the heads-up about the contact info! I understand it’s a time commitment, and I don’t want to add more to your plate. Thanks again for your effort and for being open with me.
